src/Functions/Temperature.cpp

```diff
@@ -23,6 +23,9 @@ void Temperature::renderFrame(Bitmap &frame, int editedValueIndex, int blinkingC
     TRACE << "Get temperature";
     float temp = clock().rtc()->temperature();
 
+    // RTC Temperature calibration 
+    temp = round(temp * 10)/10 + RTC_TEMP_CALIB;
```

Comment:
Any reason why the `round(temp * 10)/10` is needed and it's not just:
```suggestion
    temp += RTC_TEMP_CALIB;
```

This line should also rather be moved 3 lines below, after it has been determined that getting the temperature was successful so that `std::isnan(temp)` is false.